kubernetes/samples/scripts/common/wdt-utility.sh

```diff
@@ -0,0 +1,389 @@
+#!/bin/bash
+# Copyright (c) 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# Description:
+#
+#   This script contains functions for installing WebLogic Deploy Tool (WDT) and
+#   WebLogic Image Tool (WIT), and for running WDT.
```

Comment:
Script file name could be more  description, it says 'wdt' but also helps with 'wit'.  So maybe 'wdt-and-wit-utilities.sh'?